patroni/__init__.py

```diff
@@ -138,18 +139,41 @@ def shutdown(self):
 
 
 def patroni_main():
+    logdir = os.environ.get('PATRONI_FILE_LOG_DIR', None)
+    logsize = os.environ.get('PATRONI_FILE_LOG_SIZE', 25000000)
```

Comment:
I would suggest moving `logsize` and `lognum` into else branch (just to reduce scope of variables).